src/JabbR-Core/Properties/Chat.githubissues.css

```diff
@@ -0,0 +1,262 @@
+﻿.new-comments .comment {
```

Comment:
This was all previously under content/themes/default, lets keep out of properties (how did it end up here?)